tests/test_photoCal.py

```diff
@@ -210,7 +182,9 @@ def testZeroPoint(self):
         self.assertGreater(len(self.fitdiff), 50)
         # Tolerances are somewhat arbitrary; they're set simply to avoid regressions, and
         # are not based on we'd expect to get given the data quality.
-        self.assertLess(np.mean(self.fitdiff**2)**0.5, 0.07)    # rms difference
+        lq, uq = np.percentile(self.fitdiff, (25, 75))
+        rms = 0.741*(uq - lq)
```

Comment:
`0.741` is a magic constant here: please define it?

Reply:
Added a note.

0.741 is a good number to remember. It's the conversion factor between inter-quartile range and standard deviation for a Gaussian. Very handy for getting a robust measure of the noise.